internal/service/encryptionatrestprivateendpoint/tfplugingen/generator_config.yml

```diff
@@ -15,10 +20,15 @@ resources:
 
 data_sources:
   encryption_at_rest_private_endpoint:
+    schema:
+      attributes:
+        overrides:
+          cloud_provider:
+            description: Label that identifies the cloud provider of the private endpoint.
     read:
       path: /api/atlas/v2/groups/{groupId}/encryptionAtRest/{cloudProvider}/privateEndpoints/{endpointId}
       method: GET
   # encryption_at_rest_private_endpoints:
```

Comment:
I think this needs to be uncommented now?

Reply:
plural schema is mainly relying on the singular data source schema, we can uncomment but not sure if there will be much value in re-running the generation in the current state.